plugin/core/logging.py

```diff
@@ -39,3 +39,33 @@ def exception_log(message: str, ex: Exception) -> None:
 def printf(*args: Any, prefix: str = 'LSP') -> None:
     """Print args to the console, prefixed by the plugin name."""
     print(prefix + ":", *args)
+
+
+def notify(window: Optional[sublime.Window], message: str, status: str = 'LSP: see console log…') -> None:
```

Comment:
renamed, but the defaults are for those future cases

Reply:
When it's used in the future, I'd expect that you still pass a message to this function and don't use that default text.

How about `status_message: str | None = None` and then
```python
if status_message is None:
    status_message = message
```
because a few times in the code you used the same console and status message.

Also I think now that for the console message we should add the `LSP:` prefix, to make it easy to see which lines come from LSP and because the space is not so limited in the console.

Reply:
ok, added print prefix, deduped and made it optional